roles/logstash/templates/10-filters.conf.j2

```diff
@@ -129,6 +136,31 @@ filter {
             convert => { "apdex" => "float" }
         }
 
+        # Set duration
+        if [request_time] {
+            if [request_time] < 1 {
+                mutate {
+                    add_field => { "duration" => "lt_001s" }
+                }
+            } else if [request_time] < 5 {
+                mutate {
+                    add_field => { "duration" => "lt_005s" }
+                }
+            } else if [request_time] < 20 {
+                mutate {
+                    add_field => { "duration" => "lt_020s" }
+                }
+            } else if [request_time] < 120 {
+                mutate {
+                    add_field => { "duration" => "lt_120s" }
+                }                
+            } else {
+                mutate {
+                    add_field => { "duration" => "over_120s" }
```

Comment:
I think you also need to add the duration to the output config

Reply:
ah, i see. added as a tag to the nginx requests output which is where it was in the datadog parser